server/src/auth/twitter.js

```diff
@@ -121,7 +121,7 @@ function twitter(horizon, raw_options) {
                 logger.error(`Bad JSON data from oauth API: ${body}`);
                 auth_utils.do_redirect(res, make_failure_url('unparseable inspect response'));
               } else {
-                horizon._auth.generate_jwt(provider, user_id, (err3, jwt) => {
+                horizon._auth.generate(provider, user_id).nodeify(err3, jwt) => {
```

Comment:
Also `make_success_url` should be passed `jwt.token`, since it is no longer a string, but an object `{ token, payload }`.  This applies to `auth/utils.js` as well.